src/components/Navbar/index.css

```diff
@@ -7,7 +7,7 @@
   box-shadow: 0px 4.00313px 10.0078px #161616;
   width: 100%;
   display: grid;
-  grid-template-columns: 3fr 5fr 1fr;
+  grid-template-columns: 5fr 5fr 1fr;
```

Comment:
A screenshot would be helpful in visualising the difference. Minor thing though.

Reply:
Ok. But this basically increased the size of the navbar to accommodate the wiki button. Without it, in some languages the wiki button was colliding with the search bar.

Reply:
Just added

Reply:
> Ok. But this basically increased the size of the navbar to accommodate the wiki button. Without it, in some languages the wiki button was colliding with the search bar. 

Alright. Looks good then.